packages/components/scroll-area/src/story/ScrollAreaStory.vue

```diff
@@ -167,10 +167,10 @@ const RECOMMENDED_CSS__SCROLLBAR__THUMB = {
 }
 
 .animated-thumb[data-state="visible"] {
-  animation: fadeIn 300ms ease;
+  animation: thumbFadeIn 300ms ease;
 }
 
 .animated-thumb[data-state="hidden"] {
-  animation: fadeOut 300ms ease;
+  animation: thumbFadeOut 300ms ease;
 }
```

Comment:
The animations get overridden by other components. I just renamed the keyframes names here, but I think using either `Vue's Transition component` to handle the animation or changing the style to `scoped` would be better.

Reply:
this is because when you mouse over the menu the story loads the styles of the other component, this is a storybook bug :D 

![image](https://github.com/oku-ui/primitives/assets/38668796/41603af1-c90d-4b62-908d-16467dbe3ea7)


this is what happens if you run over them with the mouse. So don't go over them and refresh the page if you do.

Reply:
so you can undo the css change, you just need to pay attention to this. do you need to find out if this issua has been opened in storybook or can you open an issua there?

Reply:
I changed them back to `fadeIn` and `fadeOut`.Then, I refreshed the page many times, but the animation still overridden which removes the `transform: scale(...)`!

Reply:
okay thats true, https://github.com/oku-ui/primitives/pull/375 all change css names